routes/api/v1/playlists.js

```diff
@@ -0,0 +1,24 @@
+var express = require('express');
+var router = express.Router();
+
+const fetch = require('node-fetch');
+const environment = process.env.NODE_ENV || 'development';
+const configuration = require('../../../knexfile')[environment];
+const database = require('knex')(configuration);
+
+router.post('/', (request, response) => {
+  if (request.body.title) {
+    database('playlists').insert({title: request.body.title}, 'id')
+    .then(playlist => {
+      database('playlists').where('id', playlist[0])
```

Comment:
Interested to see if we can implement this without another DB call

Reply:
Not sure at this time, but we can make a card to refactor.